### DISCUSSION TITLE
Layers: Export Types

### DISCUSSION THREAD

**Discussion by aeropagz** [nuxt/nuxt#19026] [Questions] [answered]
How can I export typescript types in a layer so that child-projects can you use them?

**Comment by Baroshem** [comment]
Hey, 

I would recommend to create a separate npm package (called for example `@your-project/types`) and then use it across all your projects :)

**Comment by aeropagz** [comment]
hm, is there a way to export the type of the layer package, or do I need a dedicated package for this?

**Comment by aeropagz** [comment, marked as answer]
Okay I solved it.
I created a types dir and includes my types there. Then I declared my types in the package.json.
Following [this](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package)

Now types can be imported from the parent package 😄
